### PR TITLE
NEUBURG-96: Add bayreuth's hostname

### PR DESCRIPTION
**Pull request by michael-markl**

### CODE REVIEW

File: src/cities/cityConfigs.js [hunk 1]
```diff
@@ -12,7 +12,7 @@ const configs = {
   }),
   bayreuthwohnraum: new CityConfig({
     cmsName: 'bayreuthwohnraum',
-    hostname: 'bayreuth.wohnen.integreat-app.de',
+    hostname: 'wohnraumboerse.bayreuth.de',
```
Comment (maxammann) [inline src/cities/cityConfigs.js:15]: I think we should wait until the DNS record is there. Maybe they want **www.**wohnraumboerse.bayreuth.de

**Review by maxammann — CHANGES_REQUESTED** [review]

File: src/cities/cityConfigs.js (hunk 1 above)
Comment (michael-markl) [reply src/cities/cityConfigs.js:15]: Hmm... true :+1:

Comment (michael-markl) [reply src/cities/cityConfigs.js:15]: funnily, neuburg don't redirect www.raumfrei.neuburg-schrobenhausen.de to raumfrei.neuburg-schrobenhausen.de... Should we do sth about that (aka remind them that they could do sth about it)?

**Review by NKlug — APPROVED** [review]

File: src/cities/cityConfigs.js (hunk 1 above)
Comment (maxammann) [reply src/cities/cityConfigs.js:15]: @Schedulaar I think we do not need to do this. What do you think, @dkehne ?

**Review by maxammann — APPROVED** [review]

**Comment by michael-markl** [comment]
Done in https://github.com/Integreat/neuburg-backend/pull/39

**Pull request closed without merging**